### PR TITLE
[Admin] Rework of order, coupon and products UI

### PR DESCRIPTION
**Pull request by pjedrzejewski**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Related tickets | -
| License         | MIT

### CODE REVIEW

**Comment by pjedrzejewski** [comment]
zrzut ekranu 2016-05-23 o 20 05 36

**Comment by peteward** [comment]
New admin certainly looks very pretty! 👍 

One small suggestion: align all financial values consistently to the right as is the 'spreadsheet standard' for readability. I would align headings centrally unless their content was left-aligned as in the `Item` column.

File: src/Sylius/Bundle/AdminBundle/Resources/views/Product/Tab/_taxes.html.twig [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div class="ui tab" data-tab="taxes">
+    <h3 class="ui dividing header">{{ 'sylius.ui.taxes'|trans }}</h3>
+    {{ form_row(form.masterVariant.taxCategory) }}
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Product/Tab/_taxes.html.twig]: This should be only displayed for simple product.

Comment (michalmarcinkowski) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Product/Tab/_taxes.html.twig]: Ok, I see https://github.com/Sylius/Sylius/pull/5095/files#diff-a2f6794c3f403a31c20687f02aeb90bfR4 :)

File: src/Sylius/Bundle/AdminBundle/Resources/views/Product/_tabs.html.twig [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{% include '@SyliusAdmin/Product/Tab/_details.html.twig' %}
+{% include '@SyliusAdmin/Product/Tab/_taxonomy.html.twig' %}
+{% include '@SyliusAdmin/Product/Tab/_attributes.html.twig' %}
+{% if product.simple %}
+{% include '@SyliusAdmin/Product/Tab/_media.html.twig' %}
+{% include '@SyliusAdmin/Product/Tab/_taxes.html.twig' %}
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Product/_tabs.html.twig]: Shouldn't these be indented?

File: src/Sylius/Component/Core/Repository/OrderRepositoryInterface.php [hunk 3]
```diff
@@ -19,6 +19,8 @@
 
 interface OrderRepositoryInterface extends BaseOrderRepositoryInterface
 {
+    public function createListQueryBuilder();
```
Comment (michalmarcinkowski) [inline src/Sylius/Component/Core/Repository/OrderRepositoryInterface.php]: Missing doc block.

**Comment by michalmarcinkowski** [comment]
Thanks!

**Pull request merged**